ui/components/app/dropdowns/network-dropdown.js

```diff
@@ -149,11 +150,9 @@ class NetworkDropdown extends Component {
     );
   }
 
-  renderCustomRpcList(rpcListDetail, provider, opts = {}) {
-    const reversedRpcListDetail = rpcListDetail.slice().reverse();
-
-    return reversedRpcListDetail.map((entry) => {
-      const { rpcUrl, chainId, ticker = 'ETH', nickname = '' } = entry;
+  renderCustomRpcList(networkConfigurations, provider, opts = {}) {
+    return Object.entries(networkConfigurations).map(([uuid, entry]) => {
```

Comment:
Nit: This isn't an `entry` anymore — it's a `networkConfiguration`

Reply:
done in latest